.github/workflows/pr_branching_stragegy_validation.yml

```
@@ -0,0 +1,28 @@
+name: Validate Branching Strategy Consistency
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  validate_branching_strategy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Validate Branching Strategy
+        run: |
```

actionlint (1.7.7)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)